designs/administrator/olm/operand-form-yaml-switching/index.md

```diff
@@ -1,26 +1,38 @@
 ---
 parent: Administrator
```

Comment:
If this is about being able to flip between YAML Editor and a Form, should this be in conventions?  If so, in admin & dev we could call out any specific instances and link back to the general convention.

Reply:
This interaction could probably be rolled into the editing and creating conventions, I agree. I will add that once this PR merges.

Reply:
I have added this new create interaction to the create convention, and do agree that now the dev console could link to it as well for 4.5, as the interaction will appear there as well.